src/core/bpf/allow_bind/meson.build

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+allow_bind_hexdump_h = custom_target(
+    'allow-bind-hexdump.h',
+    input : 'allow-bind.c',
+    output : 'allow-bind-hexdump.h',
+    command : [build_bpf_py,
+            '--clang_exec', clang.path(),
+            '--llc_exec', llc.path(),
+            '--arch', 'x86_64',
```

Comment:
Mmh this is not great - meson should be able to tell you the target architecture (host in gnu parlance). Can we do that (and same for the headers paths which have an x86_64 default) instead?

Reply:
We should determine all include paths etc. from meson too, so it honors pkgconfig and friends, and not use any hardcoded values (or defaults which might silently break for other architectures!) in python code.